site/src/analytics-integrations.ts

```diff
@@ -124,6 +160,57 @@ export class AnalyticsIntegrations extends LitElement {
     `;
   }
 
+  private _filterChange(ev: any) {
+    this._currentTablePage = 0;
+    this._filter = ev.currentTarget?.value || "";
+  }
+
+  private _clearFilter() {
+    this._currentTablePage = 0;
+    this._filter = "";
+  }
+
+  async getData() {
+    const dataKeys = Object.keys(this.data!);
+    const lastEntry = this.data![dataKeys[dataKeys.length - 1]];
+    try {
+      const response = await ((window as any).integrationsPromise ||
+        fetchIntegrationDetails());
+      if (response.ok) {
+        this._integrationDetails = await response.json();
+
+        this._integrations = Object.keys(lastEntry.integrations)
+          .map((domain) => {
+            return {
+              domain,
+              title: this._integrationDetails[domain]?.title || domain,
+              installations: lastEntry.integrations[domain] || 0,
+            };
+          })
+          .concat(
+            Object.keys(this._integrationDetails)
```

Comment:
Can't we walk over this list as this has all integrations and then check if it is in the data?